driver/bpf/types.h

```diff
@@ -44,7 +44,11 @@ struct sys_exit_args {
 #else
 struct sys_exit_args {
 	__u64 pad;
+#ifdef USE_COLLECTOR_CUSTOM_PROBES
```

Comment:
we should avoid collector references in the fork to make it easier to upstream, if we decide to. This is defined when `!defined(RHEL_RELEASE_CODE) || RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(8, 0)` which we can use as a drop-in replacement here

Reply:
Very likely this will not be accepted upstream. This structure exactly defines the tp format for the sys_exit_args with an syscall nr defined as `long`. For the specific tp formats used by collector, the format defines the syscall nr as `int`. This actually is very specific to how collector consumes falco. However, falco is designed to use sys enter/exit only.

Reply:
Yeah, makes sense. I still think we'd be better of using the long-form definition here, partly because of the reference to collector, but also that `USE_COLLECTOR_CUSTOM_PROBES` is defined outside of this project, and feels quite fragile.

Reply:
yes, it would decouple it. @kcrane could you use the long version here as well. It might be also considerable to remove `USE_COLLECTOR_CUSTOM_PROBES` in the bpf probe as it seems to be used in place only.

Reply:
resolved in https://github.com/kcrane/falcosecurity-libs/commit/c79470d948654f272e67965014689af6c65b03c0 and last reference to USE_COLLECTOR_CUSTOM_PROBES removed from https://github.com/kcrane/collector/commit/04a208802a9bec1159e32c31006af01b61de8bd5